packages/flutter/lib/src/animation/animation_controller.dart

```diff
@@ -170,6 +197,17 @@ class AnimationController extends Animation<double>
   /// identifying animation controller instances in debug output.
   final String debugLabel;
 
+  /// The behavior of the animation or simulation when animations are disabled.
```

Comment:
If this property isn't just about the AnimationController, it would help to explain how it affects anything else.